spk/owncloud/src/service-setup.sh

```diff
@@ -5,6 +5,9 @@ WEB_DIR="/var/services/web_packages"
 if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ];then
     WEB_DIR="/var/services/web"
 fi
+if [ -z "${SYNOPKG_PKGDEST_VOL}" ]; then
+	SYNOPKG_PKGDEST_VOL="/volume1"
+fi
```

Comment:
variable SYNOPKG_PKGDEST_VOL is always predefined by DSM

Reply:
the same applies to the wizard *.sh files

Reply:
> the same applies to the wizard *.sh files

@hgy59 Unfortunately, I don't believe this is true in reality. I had previously shown from my own tests under https://github.com/SynoCommunity/spksrc/pull/5619#discussion_r1134516271 that the `SYNOPKG_PKGDEST_VOL` variable is only present during upgrade on DSM 6 and under upgrade and uninstall under DSM 7. This is why these variables were hard coded with fallback values if not defined.

Reply:
my analysis on 30.05.2021 with DSM 6.2-25556 and DSM 7.0-41222 showed that `SYNOPKG_PKGDEST_VOL` is defined the in the following context (without difference between DSM 6 and DSM 7):

install:
- validate_preinst
- service_preinst
- service_postinst
- service_prestart

uninstall:
- service_poststop
- validate_preuninst
- service_preuninst
- service_postuninst

upgrade:
- service_poststop
- validate_preupgrade
- service_preupgrade
- service_postupgrade
- validate_preuninst
- validate_preinst
- service_preuninst
- service_postuninst
- service_preinst
- service_postinst
- service_prestart

_I cannot imagine that this behaviour is different with the actual versions_